Makefile

```diff
@@ -185,31 +185,21 @@ package: sdist check-sdist-local manylinux-wheels check-wheel-local
 
 target_dir := "./tmp-lambda"
 install-lambda-modules:
-	@if [ -f ./dist/solarwinds_apm_lambda_${platform}.zip ]; then \
-		echo -e "Deleting old solarwinds_apm_lambda_${platform}.zip"; \
-		rm ./dist/solarwinds_apm_lambda_${platform}.zip; \
+	@if [ -f ./dist/solarwinds_apm_lambda.zip ]; then \
+		echo -e "Deleting old solarwinds_apm_lambda.zip"; \
+		rm ./dist/solarwinds_apm_lambda.zip; \
 	 fi
 	rm -rf ${target_dir}
 	@echo -e "Creating target directory ${target_dir} for AWS Lambda layer artifacts."
 	mkdir -p ${target_dir}/python
 	@echo -e "Install setuptools"
-	@set -e; for PYBIN in cp38-cp38 cp39-cp39 cp310-cp310 cp311-cp311 cp312-cp312; do /opt/python/$${PYBIN}/bin/pip install setuptools; done
+	/opt/python/cp312-cp312/bin/pip install setuptools
```

Comment:
Is it just a name and doesn't bind to python 3.12?

Reply:
This does bind to Python 3.12 to do the pip install. Else it would use the container's default Python 3.8, which is probably fine but OTel Python will drop Python 3.8 support soon.

Reply:
Changed to Python 3.9 as discussed.